gno.land/cmd/gnoland/start.go

```diff
@@ -29,6 +29,15 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+var startGraphic = fmt.Sprintf(`
+                      __                __
+   ____ _____  ____  / /___ _____  ____/ /
+  / __ %c/ __ \/ __ \/ / __ %c/ __ \/ __  / 
+ / /_/ / / / / /_/ / / /_/ / / / / /_/ /  
+ \__, /_/ /_/\____/_/\__,_/_/ /_/\__,_/   
+/____/                                    
+`, '`', '`')
+
```

Comment:
🤩  We need one of these on GnoDev too!

Can we have (full) JSONL log? If yes, we need to disable the banner to prevent line parsing problems on json logFormat. Or maybe you could play with different std output to mitigate the issue?

Reply:
🤦‍♂️ 

Good catch 💯 

Conditionally printed it:
[f554f5f](https://github.com/gnolang/gno/pull/1916/commits/f554f5fbb29f1d128fbf27c733305110925aede1)